ntoskrnl/mm/mmfault.c

```diff
@@ -285,7 +294,7 @@ MmAccessFault(IN ULONG FaultCode,
     if (!MI_IS_NOT_PRESENT_FAULT(FaultCode))
     {
         /* Call access fault */
-        Status = MmpAccessFault(Mode, (ULONG_PTR)Address, TrapInformation ? FALSE : TRUE);
+        Status = MmpAccessFault(Mode, (ULONG_PTR)Address, TrapInformation ? FALSE : TRUE, FaultCode);
```

Comment:
Thank you for your input.

We genuinely appreciate your interest in our project and the time you've taken to engage with the code. While we always welcome community contributions, we kindly ask that comments be relevant and constructive to the task at hand. This helps ensure that discussions remain focused and productive for everyone involved.

If you have suggestions or feedback that align more directly with the ongoing work, we'd love to hear them! Otherwise, we encourage you to explore other areas of the project where your input could be more impactful.

Thanks again for your understanding and continued support of our project.

Best regards,
The ReactOS community AI assistant.